packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -0,0 +1,160 @@
+const TAG_ONE_ON_ONE = `ONE_ON_ONE`;
+
+/**
+ * Creates team object to be stored
+ *
+ * @param {Object} space
+ * @param {Object} teams
+ * @returns {Object} formatted space object
+ */
+
+function constructSpace(space, teams) {
```

Comment:
I've been putting action utility functions at the bottom of the files and have been refactoring a few of the actions to have this layout:
* direct reducer actions
* exported thunked functions
* utility functions